slick.grid.js

```diff
@@ -1860,6 +1860,20 @@ if (typeof Slick === "undefined") {
       stringArray.push("</div>");
     }
 
+    function htmlEscape(inputString) {
+      inputString = inputString.replace(/[<|>|&|"]/g, function (match) {
+        switch (match) {
+          case '<': return '&lt;';
+          case '>': return '&gt;';
+          case '&': return '&amp;';
+          case '"': return '&quot;';
+          case '\'': return '&#39;';
```

Comment:
While you're at it - please look at the ADS version of this function too since it has the same "missing semicolon" problem, which probably means that it isn't escaping single quotes correctly at the moment...